### PR TITLE
Add private-prow-configs-mirror tool in autoconfigbrancher pipeline

### PR DESCRIPTION
**Pull request by droslean**
/cc @petr-muller

### CODE REVIEW

**Comment by droslean** [comment]
/hold

Holding. Need to merge https://github.com/openshift/release/pull/7685 first

File: cmd/autoconfigbrancher/main.go [hunk 1]
```diff
@@ -191,6 +191,12 @@ func main() {
 
 	commitIfNeeded(fmt.Sprintf("%s --from-dir ./ci-operator/config --to-dir ./ci-operator/jobs", "ci-operator-prowgen"), author)
 
+	cmd = "/usr/bin/private-prow-configs-mirror"
+	args = []string{"--release-repo-path", "."}
+	run(cmd, args...)
+
+	commitIfNeeded(fmt.Sprintf("%s --release-repo-path .", "private-prow-configs-mirror"), author)
```
Comment (petr-muller) [inline cmd/autoconfigbrancher/main.go]: The useless `Sprintf` triggers me, but I think I'll just file a followup PR that will fix all of them.

**Pull request merged**